if not os.path.exists(path):

```diff
@@ -915,7 +915,7 @@ def map_filename(filename):
         return(paths)
 
     def save_text_file(self, positive, negative, seed, modelname, path, filename):
-        
+        path = self.output_dir 
```

Comment:
This just causes it to ignore whatever path is passed to the function and assign it to the default.